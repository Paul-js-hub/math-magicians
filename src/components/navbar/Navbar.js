import React from 'react';
import { Link } from 'react-router-dom';
import './navbar.css';

const Navbar = () => (
  <div>
  <nav className="navbar">
    <div className="logo">Math Magicians</div>
    {/*<ul className="nav-links">*/}
      <div className="menu">
        <Link to="/">Home</Link>
        <Link to="/calculator">Calculator</Link>
        <Link to="/quotes">Quotes</Link>
      </div>
    {/*</ul>*/}
  </nav>
  </div>
);

export default Navbar;
