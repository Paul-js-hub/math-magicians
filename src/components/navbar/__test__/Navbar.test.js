import { render, screen, fireEvent} from '@testing-library/react';
import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';
import Navbar from '../Navbar.js';

describe('Test of navbar', () => {
  it('test if the navbar is rendered', () => {
    const navbar = renderer
      .create(<Router><Navbar /></Router>)
      .toJSON();
    expect(navbar).toMatchSnapshot();
  });
});