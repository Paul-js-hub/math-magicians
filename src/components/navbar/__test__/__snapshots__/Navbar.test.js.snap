// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of navbar test if the navbar is rendered 1`] = `
<div>
  <nav
    className="navbar"
  >
    <div
      className="logo"
    >
      Math Magicians
    </div>
    <div
      className="menu"
    >
      <a
        href="#/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        href="#/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
      <a
        href="#/quotes"
        onClick={[Function]}
      >
        Quotes
      </a>
    </div>
  </nav>
</div>
`;
